easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-foss-2015a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'SPIDER'
+version = '22.10'
+
+homepage = 'http://spider.wadsworth.org/spider_doc/spider/docs/spider.html'
+description = """SPIDER (System for Processing Image Data from Electron microscopy and Related fields) is
+ an image processing system for electron microscopy."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'static': True}
+
+source_urls = ['http://spider.wadsworth.org/spider_doc/spider/download/']
+sources = ['spiderweb.%(version)s.tar.gz']
+
+patches = [
+    'SPIDER-%(version)s_fix-syntax-ifort.patch',
```

Comment:
probably, yes, it's actual (syntax) errors in the code

I reported these upstream btw.